package.json

```diff
@@ -51,14 +50,15 @@
     "@demos-europe/demosplan-ui": "^0.3.17",
     "@demos-europe/dp-consent": "^1.1.2",
     "@efrane/vuex-json-api": "0.0.38",
-    "@masterportal/masterportalapi": "2.22.0",
+    "@masterportal/masterportalapi": "2.35.0",
```

Comment:
Do we need to adapt further things when bumping this?

Reply:
next to the blob-thing from the other PR, nothing I could find